app/routes/planet_routes.py

```diff
@@ -0,0 +1,35 @@
+from flask import Blueprint, abort, make_response
+from app.models import planets
+
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+@planets_bp.get("")
+def get_all_planet():
+    result_list = []
+    for planet in planets:
+        result_list.append(planet.__dict__)
```

Comment:
Neat use of the dict attribute! Depending on what we're calling it on, `.__dict__` may send extra information that you don't want to share or isn't useful to a user of the API, so we would typically create our own dict so that we can control the exact data and its format that is returned.